frontend/src/SearchMenu/SearchMenu.tsx

```diff
@@ -0,0 +1,165 @@
+import { createStyles, InputBase, makeStyles, Theme } from "@material-ui/core";
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import {
+  CREATE_MURAL_API,
+  GET_ALL_COLLECTION,
+  GET_ALL_TOUR,
+} from "constants/constants";
+import SearchResultCard from "SearchResultCard/SearchResultCard";
+import { FORM } from "constants/constants";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    flexContainer: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "flexStart",
+      width: "40vw",
+      maxWidth: "500px",
+      padding: theme.spacing(3),
+    },
+    field: {
+      width: "50%",
+      margin: theme.spacing(0, 0, 0, 2),
+    },
+    title: {
+      fontSize: "170%",
+    },
+    searchBar: {
+      display: "flex",
+    },
+    icon: {
+      width: "10%",
+      height: "10%",
+      margin: theme.spacing(0, 2, 0, 2),
+    },
+    noResults: {
+      textAlign: "center",
+      fontSize: "170%",
+    },
+  })
+);
+
+interface ISearchMenuProps {
+  handleMuralClick: (lat: number, long: number) => void;
+  handleCancel: () => void;
+}
+
+function SearchMenu(props: ISearchMenuProps) {
+  const styles = useStyles();
+  const [murals, setMurals] = useState<any>([]);
+  const [tours, setTours] = useState<any>([]);
+  const [collections, setCollections] = useState<any>([]);
+  const [displayedMurals, setDisplayedMurals] = useState<any>([]);
+  const [displayedTours, setDisplayedTours] = useState<any>([]);
+  const [displayedCollections, setDisplayedCollections] = useState<any>([]);
+  const [searchQuery, setSearchQuery] = useState<string>("");
+
+  /**
+   * fetch all collections/tours/murals upon component load LOL!
+   */
+  useEffect(() => {
+    axios
+      .get(CREATE_MURAL_API)
+      .then((response) => {
+        if (response.data) setMurals(response.data.rows);
+      })
+      .catch((err) => console.log(err));
+    axios
+      .get(GET_ALL_COLLECTION)
+      .then((response) => {
+        if (response.data) setCollections(response.data.collections);
+      })
+      .catch((err) => console.log(err));
+    axios
+      .get(GET_ALL_TOUR)
+      .then((response) => {
+        if (response.data) setTours(response.data.tours);
+      })
+      .catch((err) => console.log(err));
+  }, []);
+
+  /**
+   * Filters a list of items matching the current search query
+   * @param collection a list of items to filter
+   * @returns a list of items matching the search query
+   */
+  function filterItems(collection: any[]): any[] {
+    return collection.filter(function (item: any) {
+      return item.name.toLowerCase().includes(searchQuery.toLowerCase());
+    });
+  }
+
+  /**
+   * Filter search results when enter is pressed
+   * @param e button click event
+   */
+  function handleEnter(e: any) {
```

Comment:
nice, this solves an undocumented bug that was causing the page to refresh on enter